_blog/2024-02-09-using-clerk-in-rust.mdx

```diff
@@ -0,0 +1,1014 @@
+---
+title: "Using Clerk authentication in Rust"
+description: This article is on how to use Clerk authentication in Rust based back-end applications and little bit of an extension on integration of shared database using Shuttle. 
+author: sourabpramanik
+tags: [rust, clerk, vite, guide]
+thumb: 
+cover: 
+date: 
+---
+
+In this article, I have shared how I built an Issue tracking application using Vite for the front-end, and [Shuttle](https://www.shuttle.rs/) and [Actix web](https://actix.rs/) for the back-end. In addition to that I have also used [Clerk](https://clerk.com/) for authentication in the front-end and protecting the endpoints in the back-end.
```

Comment:
@sourabpramanik If it isn't too much of a hassle, would be good to replace the "I's" with "we'll be.." in the whole introduction as it encompasses both you and the readers reading it. So eg. "In this article, we'll be looking at how to build an Issue tracking application.."

Reply:
Right makes sense